exercises/beer-song/canonical-data.json

```diff
@@ -11,36 +11,36 @@
             {
               "description": "first generic verse",
               "property": "recite",
-              "start verse": 99,
-              "end verse": 99,
+              "startBottles": 99,
+              "endBottles": 99,
               "expected": "99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n"
             },
             {
               "description": "last generic verse",
               "property": "recite",
-              "start verse": 3,
-              "end verse": 3,
+              "startBottles": 3,
+              "endBottles": 3,
               "expected": "3 bottles of beer on the wall, 3 bottles of beer.\nTake one down and pass it around, 2 bottles of beer on the wall.\n"
             },
             {
               "description": "verse 2",
```

Comment:
Since we are not strictly talking verses anymore, perhaps this should be:
```text
"description": "2 bottles of beer",
```
or something to the affect.

Similar changes would be needed for the other cases.

Reply:
Updated!